### ISSUE TITLE
Update Migration for Proto3

### ISSUE THREAD

**Issue by tiger5226** [lbryio/lbryschema.go#6]
Once the proto 3 is ready this needs to be updated.

**Comment by tiger5226** [comment]
This is not a duplicate. This library's migrations are used by chainquery. In order for chainquery to support the new proto3 changes. This library need to be updated to leverage the latest types branch.

**Comment by tiger5226** [comment]
I also spoke with @jackrobison and he said he has no dependencies that upgrading this would break. I started this work, yesterday but there were inconsistencies that I brought up in slack at which point I found out that there is another PR with changes to the schema. Since the schema dictates the changes required for Chainquery I need that to coalesce first. Also it isn't too smart to have nested dependencies leveraging a PR branch.

**Comment by tiger5226** [comment]
A quick update. I got pretty far with the migrations. After speaking with @tzarebczan and @eukreign, @tzarebczan confirmed @lyoshenka was ok with the PR, and @eukreign said it could be merged. The library files were not generated from all the proto file changes, so I did build it for the PR, but did not merge it. Right now I am work off of the PR branch under the assumption it will not be changing significantly.
